deps/graph-parser/src/logseq/graph_parser/test/docs_graph_helper.cljs

```diff
@@ -0,0 +1,160 @@
+(ns logseq.graph-parser.test.docs-graph-helper
```

Comment:
This is a helpful test ns as it used by frontend.handler.repo and logseq.graph-parser to ensure they parse the docs graph the same